docs/man/man1/oc-create-clusterresourcequota.1

```diff
@@ -108,7 +108,7 @@ Cluster resource quota objects defined quota restrictions that span multiple pro
 .RS
 
 .nf
-  # Create an cluster resource quota limited to 10 pods
+  # Create a cluster resource quota limited to 10 pods
   oc create clusterresourcequota limit\-bob \-\-project\-label\-selector=openshift.io/requester=user\-bob \-\-hard=pods=10
```

Comment:
I think the example should state project-annotation-selector considering openshift.io/requester is an annotation not label.